content/ri/memory-optimizations.md

```diff
@@ -69,7 +69,6 @@ You can follow one of these three steps to reclaim the memory:
 1. Restart your redis-server
 1. Increase memorysamples in redis conf. (default is 5, max is 10) so that expired keys are reclaimed faster.
```

Comment:
Maybe we can change `memorysamples` to `maxmemory-samples` by the way.
There is another one below.

```suggestion
1. Increase maxmemory-samples in redis conf. (default is 5, max is 10) so that expired keys are reclaimed faster.
```